cmd/ui.go

```diff
@@ -185,35 +263,46 @@ func showProgress(ctx context.Context, conf Config, execScheduler *local.Executi
 	//description in the TODO message in cmd/root.go)
 	if stdoutTTY && !noColor {
 		updateFreq = 100 * time.Millisecond
-		outMutex.Lock()
-		stdout.PersistentText = progressBarsPrint
-		stderr.PersistentText = progressBarsPrint
-		outMutex.Unlock()
-		defer func() {
-			outMutex.Lock()
-			stdout.PersistentText = nil
-			stderr.PersistentText = nil
```

Comment:
I think this removal is causing the flickering and cursor jumpiness when running in verbose mode on this branch, which doesn't happen on `new-schedulers`.

I initially removed it as I didn't notice any flickering without it, but wasn't focusing on verbose mode.

I just hope it's this and not more cursor positioning problems... :see_no_evil: Will look into it next week.

Reply:
Yeah, definitely looks smoother on d2141571.